build/script/steamcmd

```diff
@@ -1,7 +1,3 @@
 #!/usr/bin/env bash
 
-if [ `whoami` != "$STEAM_USER" ]; then
-	exec gosu $STEAM_USER $0 "$@"
-fi
-
```

Comment:
For example, run apache as root is very difficult. It always changes to `wwwdata`. 
Anyway, go ahead. Remember to add the `USER` command